pkgs/servers/mautrix-signal/default.nix

```
@@ -37,6 +37,14 @@ python3.pkgs.buildPythonPackage rec {
       --replace "mautrix>=0.16.0,<0.17" "mautrix>=0.16.0"
   '';
 
+  patches = [
+    (pkgs.fetchpatch {
```

```suggestion
    (fetchpatch {
```

then add `fetchpatch` after `fetchFromGitHub` in the first line.